sql/core/src/main/scala/org/apache/spark/sql/SQLContext.scala

```diff
@@ -420,8 +420,10 @@ class SQLContext private[sql](val sparkSession: SparkSession)
    * converted to Catalyst rows.
    */
   private[sql]
-  def internalCreateDataFrame(catalystRows: RDD[InternalRow], schema: StructType) = {
-    sparkSession.internalCreateDataFrame(catalystRows, schema)
+  def internalCreateDataFrame(catalystRows: RDD[InternalRow],
```

Comment:
nit: The correct code style for multiline param definition is 
```
def function(
    param1: type1,      // double indent, i.e. 4 spaces
    param2: type2)    
```
See the indentation section in http://spark.apache.org/contributing.html